requirements.txt

```diff
@@ -1,5 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
-#rpm-python
+# This can not be installed from pip, on Fedora you need to install python3-rpm.
+# https://github.com/rpm-software-management/rpm/tree/master/python/rpm
+#python3-rpm
```

Comment:
Don't forget about python2 version of the package, previously called `rpm-python`, now `python2-rpm`.
Also, since `python3-rpm` provides `rpm3-python` and `python2-rpm` provides `rpm-python`, `dnf install rpm-python` will work on F24 as well as on F25 and F26, but `dnf install python2-rpm` won't work on F24. So I think it would be better to wait until F24 EOL before doing this change.